src/main/java/be/seeseemelk/mockbukkit/ServerMock.java

```diff
@@ -426,7 +426,7 @@ public String getName()
 	@Override
 	public String getVersion()
 	{
-		return "0.1.0";
+		return BUKKIT_VERSION;
```

Comment:
That's a hard pass from me.
This version does not follow Bukkit's convention, see my comment here:

https://github.com/seeseemelk/MockBukkit/pull/161/files/ddeab6d986b9976cf9a2c465a74499bac213a140#diff-763d283381d2bc7fcc200b07f44349baea512d37c0bb809b7c8b8e087d7eb1b9

Reply:
Like now ?